technologies/README.md

```diff
@@ -31,6 +31,8 @@ This is known as **task planning**, and you can learn more in [this blog post](h
 
 In this workshop, you will explore task planning in the ROS 2 ecosystem using [SkiROS2](https://github.com/RobotLabLTH/skiros2).
 
+[Start here to solve the workshop problems with SkiROS2 and to additionally explore task planning.](SkiROS2/skiros2_pyrobosim_lib/README.md).
```

Comment:
If you need this you will need to add submodules to the checkout action in the Lint job 

https://github.com/ros-wg-delib/roscon24-workshop/blob/main/.github/workflows/docker.yml#L23

Reply:
I saw that this also creates a broken link when viewed online in this repo, so I force-pushed a commit that just makes it a link to the repo

Reply:
opsie. Apparently I did that on main. 1 sec